.github/workflows/Downstream.yml

```diff
@@ -26,24 +26,13 @@ jobs:
           - {user: SciML, repo: OrdinaryDiffEq.jl, group: Core}
           - {user: SciML, repo: OrdinaryDiffEq.jl, group: Interface}
           - {user: SciML, repo: DelayDiffEq.jl, group: Interface}
-          - {user: JuliaSIMD, repo: VectorizationBase.jl, group: Interface}
-          - {user: JuliaSIMD, repo: LoopVectorization.jl, group: Downstream3}
```

Comment:
why were these removed?

Reply:
They were moved to LoopVecTests because Aqua was acting funny I'll see if I can consolidate it again.